app/util/isVVCInstalled.js

```diff
@@ -0,0 +1,20 @@
+const { resolve, dirname } = require('path');
+
+let runOnce = false;
+
+module.exports.getVVCRoot = () => {
+  let vvcRoot = false;
+
+  try {
+    vvcRoot = resolve(dirname(require.resolve('@financial-times/vvc/package.json')), 'src');
+  } catch (e) {
+    if (!runOnce) {
+      console.info(
+        'VVC not installed. To install, ensure you have your GitHub SSH key added to the SSH agent, then npm install'
```

Comment:
Might as well add a link to the Gitihub documentation so people know how to do that
```suggestion
        'VVC not installed. To install, ensure you have your GitHub SSH key added to the SSH agent (https://docs.github.com/en/enterprise/2.17/user/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent), then npm install'
```